docs/notebooks/inequality_constraints.pct.py

```diff
@@ -262,6 +277,93 @@ def batch_efi(at):
     mask_fail=batch_mask_fail.flatten()
 )
 
+# %% [markdown]
+# ## Multiple constraints
+#
+# We'll now show how to use a reducer to combine multiple constraints. We start by adding an output to our observer, and create a set of three models.
+
+
+class Sim2(Sim):
+    threshold2 = 0.5
+
+    @staticmethod
+    def constraint2(input_data):
+        x, y = input_data[:, -2], input_data[:, -1]
+        z = tf.sin(x) * tf.cos(y) - tf.cos(x) * tf.sin(y)
+        return z[:, None]
```

Comment:
Maybe write out the whole of this class rather than inherit the objective and constrain1 defined above. That might confuse the basic user?